docs/content/functions/coll.md

```diff
@@ -200,6 +200,51 @@ $ gomplate -i '{{ .books | jsonpath `$..works[?( @.edition_count > 400 )].title`
 [Alice's Adventures in Wonderland Gulliver's Travels]
 ```
 
+## `coll.JQ`
+
+**Alias:** `jq`
+
+Filters an input object or list using the JQ language implemented by gojq.
+
+Any object or list may be used as input. The output depends somewhat on the expression; if multiple items are matched, an array is returned.
+
+JQ filter expressions can be tested at https://jqplay.org/
+
+[jq Manual]: https://stedolan.github.io/jq/manual/
+[gojq library]: https://github.com/itchyny/gojq
+[gojq differences to jq]: https://github.com/itchyny/gojq#difference-to-jq
```

Comment:
these look different from the input - can you re-generate with `make docs/content/functions/coll.md` (after addressing other comments in the doc source)?